hyundai_kia_generic.dbc

```diff
@@ -1646,4 +1646,4 @@ VAL_ 1157 LFA_SysWarning 0 "no_message" 1 "switching_to_hda" 2 "switching_to_scc
 VAL_ 1157 HDA_Icon_State 0 "no_hda" 1 "white_hda" 2 "green_hda";
 VAL_ 1157 HDA_SysWarning 0 "no_message" 1 "driving_convenience_systems_cancelled" 2 "highway_drive_assist_system_cancelled";
 
-CM_ "BO_ E_EMS11: With the exception of the Optima Hybrid, all hybrid and plug-in hybrids use this gas signal: CR_Vcu_AccPedDep_Pc. All EVs (and OptimaH) use the Accel_Pedal_EV_Pc signal. Both are 0 to 100%";
+CM_ "BO_ E_EMS11: With the exception of the Optima Hybrid, all (plug-in) hybrids use this gas signal: CR_Vcu_AccPedDep_Pos. All EVs (and OptimaH) use the Accel_Pedal_Pos signal.";
```

Comment:
Might be better to keep the phrasing more generic incase there ends up being more outliers like the Optima Hybrid

Reply:
How's that? Make it point to values.py so you can see which cars use the hev signal